### PR TITLE
Issue 98 created transaction component

### LINKED ISSUE #98: Add all actions (transaction) history
**Task**
Create component that lists all the actions with transactions user made in the Giveth system. This table should contain:
 - Link to the entity - DAC/Campaign/Milestone
 - Link to ethereum transaction
 - Date of the transaction
 - Transaction price

screen shot 2018-07-03 at 15 25 12

### PR DESCRIPTION
**Pull request by emilcecarlisa**
This fixes #98 on the front end, though it still needs appropriate property names. Shouldn't be merged until backend properties are updated (will put the link to PR here).

### CODE REVIEW

**Comment by vojtechsimetka** [comment]
Deploy preview for *giveth-dapp* processing.

Built with commit 8d62f660703d721ce181914339ce91abdb825b19

https://app.netlify.com/sites/giveth-dapp/deploys/5b4d2918c965927a6a96ec3f

**Comment by GriffGreen** [comment]
is this the pr that adds some anonymity to the dapp?

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 
 > DApp - Donation Application for charitable giving without losing ownership
 
+Hey Emilce! This is working!
```
Comment (vojtechsimetka) [inline README.md]: Not sure this should stay :)

File: src/components/views/AllTransactions.jsx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    const currentUserAddress = this.props.currentUser.address;
+    console.log(currentUserAddress);
+    const transactions = await feathersClient.service('transactions').find({
+      query: {
+        address: currentUserAddress,
+      },
+    });
+
+    this.setState({ transactions });
+    console.log(this.state.transactions);
```
Comment (vojtechsimetka) [inline src/components/views/AllTransactions.jsx]: Please remove as well.

Comment (emilcecarlisa) [reply src/components/views/AllTransactions.jsx]: Does this mean you don't want the front end component called AllTransactions and that you're only interested in the back end PR?

**Comment by cbrenn** [comment]
@GriffGreen we moved away from that because of how giveth was structured when we were working on this. Campaigns are not tied to communities, so it woukd be difficult to anonimize a campaign or restrict access to a community based on previous donations to that community. We could anonimize campaigns, if that is a design that has interest.

**Comment by emilcecarlisa** [comment]
Hi all- Can we get some guidance on this PR? Sorry it took so long to follow up but we resolved the conflict in UserWallet. @vojtechsimetka Is there anything that we should do to get this merged?

**Comment by vojtechsimetka** [comment]
Closed on accident

**Comment by emilcecarlisa** [comment]
Anything we can do to help close this?

**Comment by vojtechsimetka** [comment]
@emilcecarlisa we are going to reward you the bounty and use parts of your PR for our implementation. Sadly the requirements changed dramatically.

@geleeroyale can we do that?

**Comment by emilcecarlisa** [comment]
Best news today!

**Comment by GriffGreen** [comment]
this would allow a transparent user history.... its a really important feature!! but its not done yet... and the work here pretty much has to be redone :-( OOPS We were going to rewrite feathers..

Either way! We want to pay you out for your work @emilcecarlisa

**Comment by emilcecarlisa** [comment]
Hi @geleeroyale, we're ready to initiate payout

**Comment by geleeroyale** [comment]
Great! Looks good and I will try to merge and initiate the payout once openbounty.status.im is back online. The site is not showing **any** data anymore. I will keep you updated.

**Comment by geleeroyale** [comment]
This is not an empty PR - I do not dare to merge this.

If its just about the bounty payout, please make an empty PR referencing #98.

**Comment by emilcecarlisa** [comment]
@geleeroyale, I think I did this right. Sorry for the confusion.

**Comment by GriffGreen** [comment]
NOT A DAPP ISSUE... ITS A STATUS OPEN BOUNTY ISSUE :-D

**Comment by arbreton** [comment]
Will reopen when work is resumed on this module

**Pull request closed without merging**